.tx/config

```diff
@@ -1,43 +1,43 @@
 [main]
 host = https://www.transifex.com
 
-[bahmni.admin-app]
+[o:openmrs:p:bahmni:r:admin-app]
```

Comment:
Is this provided by transifex? What do they indicate?

Reply:
I have tried without any changes in the first place, but was not able to find the resource. came across the documentation https://developers.transifex.com/docs/using-the-client#adding-resources-to-configuration